examples/create_averaged_submissions_from_local_disk.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from typing import List
+
+import typer
+
+from embeddings.data.io import T_path
+from embeddings.defaults import SUBMISSIONS_PATH
+from embeddings.evaluator.evaluation_results import Task
+from embeddings.evaluator.submission import AveragedSubmission, Submission
+
+app = typer.Typer()
+
+
+def run(
+    dataset: str = typer.Option(..., help="Dataset"),
+    model: str = typer.Option(..., help="Model"),
+    task: Task = typer.Option(..., help="Task type"),
```

Comment:
shouldn't we use `typer.Argument` for required fields?

Reply:
it is not obligatory. Arguments are used to pass like `main.py some_value`, while options `main.py --name some_value`. 
According to docs: 
> We said before that by default:

>CLI options are optional
>CLI arguments are required
>Again, that's how they work by default, and that's the convention in many CLI programs and systems.

>But you can change that.

>**In fact, it's very common to have optional CLI arguments, it's way more common than having required CLI options.**
https://typer.tiangolo.com/tutorial/arguments/optional/

Reply:
Yes, I am aware of this and I also referred to the documentation. I believe that you have used the required CLI option here in the code and that is why I added my comment.

Reply:
From my understanding:
- required CLI option is `typer.Optional(...)`
- optional CLI argument is `typer.Argument("some_str")`